src/data/battle_moves.h

```diff
@@ -12753,29 +12753,30 @@ const struct BattleMove gBattleMoves[MOVES_COUNT_Z] =
     {
         #if B_UPDATED_MOVE_DATA >= GEN_9
```

Comment:
...This move has only been in a main series game since Gen 9? Why is this needed? (PLA is not considered a main series game in the same sense as all of the other ones).

Reply:
Well, [Nintendo does seem to consider it a main game](https://www.pokemon.co.jp/game/).

Regardless, as it's explained in some of the comments at `include/config/battle.h`, we're considering LA a Gen. 8 Game *(so we can have a configuration like this one for Triple Arrows because adding sub-version like `GEN_9_LA` labels was rejected)*, and in PLA, Triple Arrows had a power of 50 and a base amount of PP of 15.

"Why is it [the config for Triple Arrows] needed?" It isn't. I highly doubt someone will want to make use of Triple Arrows' LA data which makes the move weaker, but I have to cover my bases. We have configurations for a lot of different things, and the different aspects of move data is one of them.

I'm just going with the flow, and for better or worse, the expansion wants to have configurations for every little thing possible.

Reply:
it might be better in that case to invert the condition and make a special define for PLA moves (such as `#ifdef PLA_MOVE_EFFECTS`) since they don't belong to any gen. That way if someone wants the moves to do what they did in PLA, they'll get all moves that existed in PLA to do what they did.

Reply:
> since they don't belong to any gen

PLA *is* Gen 8, but I get what you mean. Since we're not planning on adding PLA's combat mechanics, having the PLA data for these moves and not others is a bit redundant.

Reply:
How do you want me to proceed then, @AsparagusEduardo?

Should I just remove the move config for the PLA moves?